data/org.gnome.shell.gschema.xml.in

```diff
@@ -158,15 +158,14 @@
         This key specifies the exact order of the icons shown in the applications launcher view.
       </description>
     </key>
-    <key name="enable-discovery-feed" type="b">
-      <default>false</default>
+    <key name="discovery-feed-languages" type="as">
+      <default>['en']</default>
       <summary>
-        Whether the discovery feed feature is enabled
+        Languages for which the discovery feed feature is enabled.
       </summary>
       <description>
-        Whether the discovery feed feature is enabled. You will need to restart
-        the shell for this setting to take effect, as it is intended
-        for product developers.
+        If the system language matches any of these languages, the
+        discovery feed will be enabled on the next shell reboot.
```

Comment:
The wording is not 100% accurate here I guess, we have it set to "en" by default so it will be enabled by default if en is the system language. Maybe:

`If the system language matches any of these languages, the discovery feed will be enabled. You will need to restart the shell for changes to this setting to take effect.`